bitbot/links.go

```diff
@@ -50,8 +50,8 @@ func isTitleElement(n *html.Node) bool {
 func traverse(n *html.Node) (string, bool) {
 	if isTitleElement(n) {
 		if n.FirstChild != nil {
-			if len(n.FirstChild.Data) > 350 {
-				return n.FirstChild.Data[:350], true
+			if len(n.FirstChild.Data) > 120 {
```

Comment:
This was increased to 350 to accommodate max Twitter names + Tweet length. 120 is an arbitrary value that doesn't correspond to anything meaningful. 

Max message length in RFC 1459 is 510 characters, so let's append `...` at 350